x/executionlayer/client/cli/tx.go

```diff
@@ -412,26 +457,53 @@ func GetCmdCreateValidator(cdc *codec.Codec) *cobra.Command {
 	}
 
 	cmd.Flags().String(client.FlagHome, DefaultClientHome, "Custom local path of client's home dir")
-	cmd.Flags().String(client.FlagFrom, "", "Bech32 encoded address (fridayxxxxxx...)")
+	cmd.Flags().String(FlagAddress, "", "Bech32 endocded address (fridayxxxxxx..)")
+	cmd.Flags().String(FlagWallet, "", "Wallet alias")
+	cmd.Flags().String(FlagNickname, "", "Nickname")
 	cmd.Flags().AddFlagSet(fsDescriptionCreate)
 	cmd.Flags().AddFlagSet(FsPk)
 
-	cmd.MarkFlagRequired(client.FlagFrom)
 	cmd.MarkFlagRequired(FlagPubKey)
 	cmd.MarkFlagRequired(FlagMoniker)
 
 	return cmd
 }
 
-// BuildCreateValidatorMsg implements for adding validator module spec
-func BuildCreateValidatorMsg(cliCtx context.CLIContext) (sdk.Msg, error) {
-	kb, err := client.NewKeyBaseFromDir(viper.GetString(client.FlagHome))
-	if err != nil {
-		return types.MsgCreateValidator{}, err
+// GetCmdEditValidator implements the create edit validator command.
+func GetCmdEditValidator(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit-validator",
+		Short: "edit an existing validator account",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			txBldr := auth.NewTxBuilderFromCLI().WithTxEncoder(auth.DefaultTxEncoder(cdc))
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
```

Comment:
Has no option for `--wallet|--address|--nickname <from>`

Reply:
Currently works well with `--from <wallet_alias>`

Reply:
https://github.com/hdac-io/friday/pull/65/commits/f7f3c1a29678962c569779ba5498358367547756